pkg/apis/service/v1alpha1/types.go

```diff
@@ -43,6 +46,27 @@ type FalcoServiceConfig struct {
 	// Configuration for custom webhook
 	// +optional
 	CustomWebhook *Webhook `json:"webhook,omitempty"`
+
+	// -------------------------------------------------------------------
+	// added due to issue #215
+
+	Rules *Rules `json:"rules,omitempty"`
+
+	Destinations *[]Destination `json:"destinations,omitempty"`
+}
+
+type Destination struct {
+	Name              string  `json:"name,omitempty"`
+	ResourceSecretRef *string `json:"resourceSecretRef,omitempty"`
```

Comment:
This should be ResourceName (i.e., the name of a resource in `.spec.resources[]` in the `Shoot`)

Reply:
We decided to name it with "secret" as it is expected to be a secret and not a configmap

Reply:
This detail is already configured in `.spec.resources[]`. Here, you just reference a resource. Think of it like a volume mount that just references the name of a volume. In the mount, you don't replicate again which kind of volume it is.